app/assets/stylesheets/setup-mode.scss

```diff
@@ -6,6 +6,10 @@ body.default-header {
   padding-top: 0;
   min-height: 100vh;
 }
+.creator-settings-fields {
+  height: 40px;
+  max-width: 480px;
+}
```

Comment:
I took the height from the Figma design and added the `max-width` as `480px` per this comment. However, I wonder if there's a better way to do this via utility classes/Crayons.